python_examples/megno/problem.py

```diff
@@ -32,7 +32,7 @@ def setup_planet(com, mass, period, M, omega, eccentricity):
     if eccentricity>0.8:
         E = math.pi
     F = E - eccentricity*math.sin(M) - M
-    for i in xrange(100):
+    for i in range(100):
```

Comment:
Right.  The only thing that wouldn't work is something like `range(5) + range(3)`, which would give a TypeError in Python 3.x but wouldn't in 2.x with the ordinary meaning of range().  So, we in principle might break some backwards compatibility, but only for things that people probably won't do anyway.